packages/cojson-transport-ws/src/BatchedOutgoingMessages.ts

```diff
@@ -0,0 +1,36 @@
+import { SyncMessage } from "cojson";
+
+export const MAX_OUTGOING_MESSAGES_CHUNK_SIZE = 10;
```

Comment:
I've set this up to 10 to stay below the 1Kb per batch.

The websockets are on TCP so we shouldn't exceed the window size, but honestly I don't remeber the math behind the window size calculation so I'm setting this under a safe value before doing more research.